autoinstrumentation/dotnet/Dockerfile

```diff
@@ -26,4 +26,8 @@ ADD https://github.com/open-telemetry/opentelemetry-dotnet-instrumentation/relea
 RUN unzip opentelemetry-dotnet-instrumentation-linux-glibc.zip &&\
     unzip opentelemetry-dotnet-instrumentation-linux-musl.zip "linux-musl-x64/*" -d . &&\
     rm opentelemetry-dotnet-instrumentation-linux-glibc.zip opentelemetry-dotnet-instrumentation-linux-musl.zip &&\
-    chmod -R go+r .
\ No newline at end of file
+    chmod -R go+r .
+
+FROM busybox
```

Comment:
Do we even need this to be buxybox, as opposed to scratch?

Reply:
Based on #1600 - yes.
What is more, other languages follow the same pattern.